src/cloud/docker/docker-containers-service.md

```diff
@@ -82,93 +66,83 @@ You can  load custom extensions in the FPM configuration by adding the configura
 
 For additional information about configuring the php environment, see the [XDebug for Docker]({{site.baseurl}}/cloud/docker/docker-development-debug.html) documentation.
 
-### Rabbitmq Container
-
-#### Container Information
+### Rabbitmq container
 
--  Name: rabbitmq
--  Docker Base Image: [rabbitmq](https://hub.docker.com/_/rabbitmq)
--  Ports Exposed: 4369,5671,5672,25672
+Container name |Docker base image | Ports exposed
+-------- | -------- | ---------------
+rabbitmq | [rabbitmq](https://hub.docker.com/_/rabbitmq) | 4369, 5671, 5672, 25672
 
-#### Container Usage
+The RabbitMQ container for {{site.data.var.mcd}} is a standard RabbitMQ container with no configuration or changes.
 
-Standard RabbitMQ Container, with no configuration or changes from Magento Cloud Docker.
+### Redis container
 
-### Redis Container
+Container name |Docker base image | Ports exposed
+-------- | -------- | ---------------
+redis | [redis](https://hub.docker.com/_/redis) | 6379
 
-#### Container Information
+The RabbitMQ container for {{site.data.var.mcd}} is a standard Redis container with no customization, no persistence, and no additional configuration.
 
--  Name: redis
--  Docker Base Image: [redis](https://hub.docker.com/_/redis)
--  Ports Exposed: 6379
+Connect to and run redis commands using the redis-cli inside the container:
 
-#### Container Usage
-A standard Redis container with no customizations, no persistence or configuration is used.
-
-Connect to and run redis commands via the redis-cli inside the container:
 ```bash
 docker-compose run redis redis-cli -h redis
 ```
 
-### TLS Container
+## Selenium container
+
+Container name |Docker base image | Ports exposed
+-------- | -------- | ---------------
+selenium |[selenium/standalone-chrome/](https://hub.docker.com/r/selenium/standalone-chrome/h) | 4444
```

Comment:
Image name should just be `selenium/standalone-chrome` (no trailing slash) and URL should not have `h` at the end.